### PR TITLE
Fixes #3565 - lastpost_subject _ CSS fix

### LINKED ISSUE #3565: Enhancement: $lastpost_subject > 25
The number of characters shouldn't be hard coded. It should be set in Forumhome or Forumdisplay settings.

Different templates, not to mention mobile screens, require this to be adaptable.

inc/functions_forumlist.php
Line 312
```
if(my_strlen($lastpost_subject) > 25)
{
	$lastpost_subject = my_substr($lastpost_subject, 0, 25)."...";
}
```

----------

Edit: Infact, can there be no hardcoded limit and use css text-overflow: ellipsis; in the template?

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #3565

CSS solution... PHP solution was rejected #4288

### CODE REVIEW

File: install/resources/mybb_theme.xml [hunk 1]
```diff
@@ -448,6 +448,14 @@ table {
 	color: #999;
 }
 
+.lastpost_subject {
+	display: inline-block;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	width: 200px;
```
Comment (Sama34) [inline install/resources/mybb_theme.xml:456]: Can't the width be `100%` or removed? I know the default theme isn't mobile friendly but we won't want to make it worse.

Comment (Eldenroot) [reply install/resources/mybb_theme.xml:456]: I dont think so... now it uses same width as td in table.

Comment (effone) [inline install/resources/mybb_theme.xml:456]: You should not fix the size here. Table column has its own width specified.
Better use `display: block;` or a `div`.

Comment (Eldenroot) [reply install/resources/mybb_theme.xml:456]: I have tried both and no success...

Comment (Sama34) [reply install/resources/mybb_theme.xml:456]: Perhaps @justinsoltesz could give us some insight?

Comment (Eldenroot) [reply install/resources/mybb_theme.xml:456]: @justinsoltesz @effone - any feedback how to solve this?

**Comment by Eldenroot** [comment]
Bump

**Comment by euantorano** [comment]
Any feedback @mybb/developers @mybb/sqa? We know that CSS changes don't get applied automatically on upgrade, but are we happy with the content of the changes? I'm okay with them personally.

**Review by Sama34 — APPROVED** [review]
Seems like it is not as easy to not use a set width, so I'm fine with it.

**Pull request merged**